content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,377 @@
 
 Last In, First Out (LIFO)
 -------------------------
 
-The same way as for FIFO, the **Last In, First Out** strategy is based on moving products based on the
-date they entered the stock. Here, a demand for some products triggers a removal rule that requests a
-transfer for the lot/serial number that has entered your stock the last.
+Similar to the :abbr:`FIFO (First In, First Out)` method, the :guilabel:`Last In, First Out (LIFO)`
+removal strategy moves products based on the date they entered a warehouse's stock.  Instead of
+removing the oldest stock on-hand, however, it targets the **newest** stock on-hand for removal.
 
-To better understand, let’s imagine three lots of screws in your warehouse. Those three have the
-following numbers: 10001, 10002, 10003, each with 10 screw boxes in it. 10001 has entered the stock
-on the 1st of June, 10002 on the 3rd of June, and 10003 on the 6th of June. A customer orders
-7 boxes on the 8th of June. With the *LIFO* strategy selected, a transfer is requested for seven
-boxes of 10003 because that lot is the last one to have entered the stock.
+Every time an order for products with the :abbr:`LIFO (Last In, First Out)` method is placed, a
+transfer is created for the lot/serial number that has most recently entered the stock (the **last**
+lot/serial number that entered the warehouse's inventory).
 
-So, basically, for every order of a product with the *LIFO* strategy used, a transfer for the last
-one to have entered the stock is requested.
+.. Warning::
+   In many countries, the :abbr:`LIFO (Last In, First Out)` removal strategy in banned, since it can
+   potentially result in old, expired, or obsolete products being delivered to customers.
 
-.. note::
-   This strategy is banned in many countries and can lead to only have old or obsolete products
-   in your stock.
+For example, imagine there are three lots of screws in the warehouse. Those three have the following
+numbers: 10001, 10002, and 10003, each with 10 boxes of screws in it.
+
+Lot 10001 entered the stock on June 1, lot 10002 on June 3, and lot 10003 on June 6. A customer
+orders seven boxes on June 8.
+
+Using the :abbr:`LIFO (Last In, First Out)` removal strategy, a transfer is requested for seven
+boxes from lot 10003 because that lot is the last one to have entered the stock.
+
+.. image:: removal/lifo-nails.png
+   :align: center
+   :alt: The detailed operations shows which lots are being selected for the picking.
+
+First Expired, First Out (FEFO)
+-------------------------------
+
+While the :abbr:`FIFO (First In, First Out)` and :abbr:`LIFO (Last In, First Out)` methods target
+products for removal based on date of entry into the warehouse, the :guilabel:`First Expired, First
+Out (FEFO)` method targets products for removal based on their assigned expiration dates.
+
+Using the :abbr:`FEFO (First Expiry, First Out)` removal strategy, every sales order that includes
```

Comment:
Thanks- I was torn because I like to be consistent with the app, and match the words so people don't get confused. I prefer "expired" (sounds better!)